scripts/k8s_deploy_kubeflow.sh

```diff
@@ -42,40 +37,29 @@ if [ $? -eq 0 ] ; then
     exit 1
 fi
 
-# Ksonnet
-wget -O /tmp/${KS_PKG}.tar.gz "${KSONNET_URL}" \
-      --no-check-certificate
-mkdir -p ${KS_INSTALL_DIR}
-tempd=$(mktemp -d)
-tar -xvf /tmp/${KS_PKG}.tar.gz -C ${tempd}
-sudo mv ${tempd}/${KS_PKG}/ks ${KS_INSTALL_DIR}
-rm -rf ${tempd} /tmp/${KS_PKG}.tar.gz
-
 # Kubeflow
 if [ ! -d ${KUBEFLOW_SRC} ] ; then
+    echo ${KUBEFLOW_URL}
     tempd=$(mktemp -d)
     cd ${tempd}
-    curl "${KUBEFLOW_URL}" | bash
+    wget ${KUBEFLOW_URL}
+    tar -xvf "kfctl_${KUBEFLOW_TAG}_linux.tar.gz"
     cd -
     sudo mv ${tempd} ${KUBEFLOW_SRC}
 fi
 
 pushd ${HOME}
-${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none
+${KUBEFLOW_SRC}/kfctl init ${KFAPP}
 cd ${KFAPP}
-${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
-pushd ks_app
-ks param set jupyter serviceType NodePort
```

Comment:
Can we update this to something like `ks param set ambassador ambassadorServiceType LoadBalancer` if the load balancer is deployed?

Does kubeflow default to exposing ambassador on a node port now (vs cluster IP before)?

Reply:
Ah yes, we should totally do that.

And yes, it appears that Kubeflow defaults to exposing Ambassador on NodePort.

Reply:
Done!